playbooks/byo/openshift-checks/adhoc.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: OpenShift health checks
+  hosts: OSEv3
```

Comment:
All the other roles that run before `openshift_health_checker` are a problem :( and not just because it's slow. I wish we could separate them out into roles that gather info and set variables versus those which perform an action. Unfortunately there are a few actions (installs) necessary even to gather info. But it could be that those aren't necessary for our use case, and we could further divide the info-gatherers into fast and slow and just stick to info that can be gathered quickly for our checks. Maybe. I don't really look forward to that refactoring but running the checks is pretty heavy like this.